Frontend/src/lib/forms/AddUserForm.svelte

```diff
@@ -19,53 +19,83 @@
     let group = [];
 </script>
 
+<div>
+    <h2>
+        Create a User
+        <hr />
+    </h2>
+    
+    <form action="" method="POST" on:submit|preventDefault={handleSubmit} class="add-form" ><!--  prevent default is used to stop the page from refresh -->
+        
 
-<form action="" method="POST" on:submit|preventDefault={handleSubmit} class="add-form" ><!--  prevent default is used to stop the page from refresh -->
-    <label>
-        Title
-        <input type="text" placeholder="" bind:value={title}>
-    </label>
-
-    <label>
-        First Name
+        <!-- Revisit labels, it seemed at default the label tags do not work with the required-field class -->
+        <p class="required-field">Title</p>
+        <input id={title} type="text" placeholder="" bind:value={title}>
+        
+        
+        <p class="required-field">First Name</p>
         <input type="text" placeholder="" bind:value={firstName}>
-    </label>
-    
-    <label>
-        Last Name
-        <input type="text" placeholder="" bind:value={lastName}>
-    </label>
 
-    <label>
-        Email Address
-        <input type="text" placeholder="" bind:value={email}>
-    </label>
+        <p class="required-field">Last Name</p>
+        <input type="text" placeholder="" bind:value={lastName}>
 
-    <label>
-        Location(s)
-        <select bind:value={location}>
-            <option value="TEST 1">TEST</option>
-            <option value="TEST 2">TEST</option>
-            <option value="TEST 3">TEST</option>
-        </select>
-    </label>
-    <small>Create New</small>
+        
+        <p class="required-field">Email</p>
+        <input id={email} type="text" placeholder="" bind:value={email}>
+        
+        
+        <label>
+            Location(s)
+            <select bind:value={location}>
+                <option value="TEST 1">TEST</option>
+                <option value="TEST 2">TEST</option>
+                <option value="TEST 3">TEST</option>
+            </select>
+        </label>
+        <small>
+            <a>Create New</a>
+            <!-- Ask about if these are intended to be hrefs or on:clicks -->
```

Comment:
It shouldn't really matter, but it's probably easier to just make this a button that looks like an href and handle an onclick

Reply:
Modified to use a button